kubernetes/auth-operator.yaml

```diff
@@ -29,6 +29,9 @@ spec:
           image: dcodetech/aws_auth_eks_crd:1.1.8
           imagePullPolicy: IfNotPresent
           name: operator
+          env:
+            - name: IGNORED_CM_IDENTITIES
+              value: "system:node:{{SessionName}},system:node:{{EC2PrivateDNSName}}"
```

Comment:
This will also prevent removing EC2 nodes not only fargate profiles, no?

I also think it's better to hardcode it and not use it as config, because it's quite critical part

Reply:
Yes to the first and agree with the second. I'll add the change to the code itself then